tableauserverclient/server/request_options.py

```diff
@@ -62,24 +62,49 @@ def apply_query_params(self, url):
         return "{0}?{1}".format(url, '&'.join(params))
 
 
-class ImageRequestOptions(RequestOptionsBase):
+class _FilterOptionsBase(RequestOptionsBase):
+    def __init__(self):
+        self.view_filters = []
+
+    def apply_query_params(self, url):
+        raise NotImplementedError()
+
+    def vf(self, name, value):
+        self.view_filters.append((name, value))
+        return self
+
+    def _append_view_filters(self, params):
+        for name, value in self.view_filters:
+            params.append('vf_{}={}'.format(name, value))
+
+
+class CSVRequestOptions(_FilterOptionsBase):
+    def apply_query_params(self, url):
+        params = []
+        self._append_view_filters(params)
+        return "{0}?{1}".format(url, '&'.join(params))
+
+
+class ImageRequestOptions(_FilterOptionsBase):
     # if 'high' isn't specified, the REST API endpoint returns an image with standard resolution
     class Resolution:
         High = 'high'
 
     def __init__(self, imageresolution=None):
```

Comment:
Side note: I thought I made this so it was `image_resolution`... maybe I never checked that fix in?